CSS_files/components/input/input.css

```diff
@@ -0,0 +1,50 @@
+.input{
+    padding: .8rem 1rem;
+    border: none;
+    border-radius: 10px;
+}
+
+.input-primary:focus,.text-area:focus{
+    outline-color:var(--primary-color) ;
+}
+
+/* .input-primary{
```

Comment:
You can remove the CSS which is not required instead of commenting them

Reply:
Done. Thank you.